IDEAS/Buildings/Components/SlabOnGround.mo

```diff
@@ -19,6 +19,10 @@ model SlabOnGround "opaque floor on ground slab"
     "Annual average outdoor temperature";
   parameter Modelica.Units.SI.Temperature TiAvg=273.15 + 22
     "Annual average indoor temperature";
+  parameter Modelica.Units.SI.Temperature T_start_ground[3]={TeAvg,TeAvg,TeAvg}
```

Comment:
I would either create a new final (and protected) parameter `nLayGro` or use `layGro.nLay` here and then set the initial value equal to `fill(TeAvg, nLayGro)`, just to make the model a slightly bit more robust.

If you create the parameter, you could also use this to vectorize the ground material parameter (`IDEAS.Buildings.Data.Materials.Ground`).

+ Naming convention (`T_start_gro`)

Reply:
Ok, good suggestion. I have modified the model accordingly.